subprojects/smoke-test/build.gradle.kts

```diff
@@ -167,6 +169,11 @@ tasks {
     }
 }
 
+if (isTeamCity) {
+    registerEmitTeamCityImportDataServiceMessageBuildService(
+        "smokeTest", "configCacheSmokeTest", "gradleBuildSmokeTest", "santaTrackerSmokeTest", "configCacheSantaTrackerSmokeTest")
```

Comment:
Could we somehow avoid hardcoding the names here? Given that we are in the smoke test plugin, we should somehow know the names already, right?

In general, I suppose we do this for tasks of type `Test`. Is there a way to find out the task type directly from the `TaskFinishEvent`?

Reply:
Nice reminder! Now I enabled this for all `Test` tasks, our windows tests will benefit from this now.